src/main/java/hw_17/Test1.java

```diff
@@ -0,0 +1,34 @@
+package hw_17;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import java.time.Duration;
+import org.junit.Assert;
+
+
+public class Test1 {
+    public static void main(String[] args) {
+        WebDriver driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("https://makeup.com.ua/ua/");
+
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(15));
+
+        WebElement searchButton = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("div.search-button")));
+        searchButton.click();
+
+        WebElement searchField = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("input#search-input")));
+        searchField.sendKeys("lipstick");
+
+        WebElement searchActivateButton = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("a.search-result__link[href='/ua/search/?q=lipstick']")));
+        searchActivateButton.click();
+
+        WebElement resultsContainer = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("a[href='/ua/search/?q=lipstick']")));
```

Comment:
Please use waiters only in places where it could help but not in all places

Reply:
done